draft-ietf-lamps-dilithium-certificates.md

```diff
@@ -925,7 +925,7 @@ supports a pre-hashing flow by splitting the operation over two
 modules. In this section, we make this "ExternalMu-ML-DSA"
 more explicit.
 
-There are two steps. First an `ExternalMu-ML-DSA.Prehash()`
+There are two steps. First an `ExternalMu-ML-DSA.Pure-Prehash()`
```

Comment:
I understand the intention, but I would rather we kept it as-is. ML-DSA is the name for pure in FIPS 204. If we were defining it for HashML-DSA we could call it that.

Reply:
Reverted this part